packages/material-ui-codemod/src/util/memoize.js

```diff
@@ -0,0 +1,12 @@
+const memoize = (func, resolver = a => a) => {
```

Comment:
`yarn add -D -W memoize-one`

It's already in node_modules from 4 other dependencies

Reply:
Unfortunately `memoize-one` is not great for this, as it only remembers the most recent arguments.  With this impl I assume cached info from parsing will stay in memory from one file being transformed to the next.  (If passing a dir to jscodeshift instead of the strange usage of find/xargs, at least)

Reply:
Code size isn't a major concern for the codemods package is it?